site/en/docs/extensions/mv3/favicon/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Fetching favicons"
+seoTitle: "Fetching with favicons in Chrome extensions"
+date: 2023-01-11
+# updated: 2023-02-14
+description: How to get a websites' favicon in a Chrome extension.
+---
+
+## Overview {: #overview }
+
+A [favicon][mdn-favicon] (short for "favorite icon") is a small icon that is displayed in the browser's address bar. Favicons are typically used to identify and differentiate websites.
+This article describes how to retrieve a website’s favicon in a Manifest version 3 extension.
+
+## Accessing a websites' favicon {: #how-to } 
+
+To retrieve the favicon of a website, you need to construct the following URL:
+
+```text
+chrome-extension://EXTENSION_ID/_favicon/?pageUrl=EXAMPLE_URL&size=FAV_SIZE
+```
+
+`EXTENSION_ID`
+: The ID of your extension.
+
+`EXAMPLE_URL`
+: The URL of the favicon's website.
+
+`FAV_SIZE`
+: The size of the favicon. For example: 16, 32, or 64.
+
+The following steps describe how to construct this URL in a Chrome extension:  
+
+### Step 1: Update the manifest {: #manifest }
```

Comment:
```suggestion
### Step 1: update the manifest {: #manifest }
```

Reply:
I'm a thousand miles from my reference library, so I can't tell you the rule number just now. The Chicago Manual of Style, which is Google corporate standard, says that text following a colon is lower case.